### PR TITLE
i18n(ko-KR): update `cms/`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)
- update `cms/`
- #9240 
- #10117

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10177 is merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
